plugins/restful/RestfulEntityBase.php

```diff
@@ -698,7 +698,18 @@ protected function setPropertyValues(EntityMetadataWrapper $wrapper, $null_missi
         $field_value = $this->propertyValuesPreprocess($property_name, $request[$public_field_name], $public_field_name);
       }
 
-      $wrapper->{$property_name}->set($field_value);
```

Comment:
@AronNovak 
can you please add an elaborated comment about this change?
thanks!

Reply:
@ordavidil added a few comments

Reply:
cool, can you also push it?   :)
thanks !

Reply:
:) Now i did

Reply:
thanks!